### ISSUE TITLE
Lates version 1.3 can not be opened

### ISSUE THREAD

**Issue by zuidwijk** [esphome/esphome-flasher#26]
<img width="532" alt="Schermafbeelding 2020-08-01 om 19 43 07" src="https://user-images.githubusercontent.com/10123063/89107144-45ee0680-d42f-11ea-9a5c-8a9a4b163ffb.png">

I'm working on a MacBook Pro 2019 with latest Catalina on it.

**Comment by JJK801** [comment]
Same issue here

**Comment by AalianKhan** [comment]
1.2 works but 1.3 doesn't

**Comment by zuidwijk** [comment]
> 1.2 works but 1.3 doesn't

Yep here too! 1.2.0 works perfectly fine. 1.3.0 gives the error which I posted.

**Comment by DavidDeSloovere** [comment]
FYI The Windows build works fine (after bypassing Smartscreen)

**Comment by philletourneau** [comment]
I can't open it either, FWIW

**Comment by dotWee** [comment]
Same here on macOS 10.15.6. Trying to open the application (`$ open ESPHome-Flasher-1.3.0-macOS/ESPHome-Flasher.app`) fails with this exception: 


> LSOpenURLsWithRole() failed with error -10810 for the file /Users/lukas/Downloads/ESPHome-Flasher-1.3.0-macOS/ESPHome-Flasher.app.

**Comment by zuidwijk** [comment]
Found it! This solves it:

`chmod +x ESPHome-Flasher.app/Contents/MacOS/ESPHome-Flasher`

```
sh-3.2# ls -l /Applications/ESPHome-Flasher.app/Contents/MacOS/
total 30792
-rw-r--r--@ 1 marcel  staff  15765270 30 jul 11:29 ESPHome-Flasher
sh-3.2#
```

**Comment by zuidwijk** [comment]
@dotWee Your error message gave me the clue :)

**Comment by zuidwijk** [comment]
@OttoWinter can you fix this in the next/upcoming version?

**Comment by dotWee** [comment]
Can confirm the application is just missing the executable flag. Thanks!

**Comment by cmriquinha** [comment]
Hey, all I know it's been a bit since yall commented on this.  I have tried many versions.  Including ESPHome-Flasher (which will not open, per initial issue) as well as NodeMCU Pyflasher v.3 and v.4 apps not seeing usb serial port.  I am a bit of a loss and can not move on until I figure it out.  Any help would be appreciated as I am still unable to flash the esp8266 running on Mac Catelina.

**Comment by suhlig** [comment]
```command
chmod +x /Applications/ESPHome-Flasher.app/Contents/MacOS/ESPHome-Flasher
```
fixes the problem.

**Comment by zuidwijk** [comment]
@cmriquinha have you tried what @suhlig and I have mentioned as solution?

```
chmod +x /Applications/ESPHome-Flasher.app/Contents/MacOS/ESPHome-Flasher
```

**Comment by jezmck** [comment]
I also had to allow the app to run in System Preferences > Security & Privacy > General (a button should appear at the bottom).

**Comment by ronnyandre** [comment]
Still an issue.

**Comment by ok-morpheus** [comment]
> ```shell
> chmod +x /Applications/ESPHome-Flasher.app/Contents/MacOS/ESPHome-Flasher
> ```
> 
> fixes the problem.

Great info.  But you guys are assuming the readers know to type that on a terminal command line.

**Comment by IZ1IVA** [comment]
This fix unfortunately doesn't work on macOS 10.13.6 (tried on two different iMacs).

**Comment by genebean** [comment]
Running the `chmod` command above via sudo in a terminal on BigSur and then allowing it to run from Security & Privacy worked for me

**Comment by hmakmur** [comment]
I can confirmed that this does not work with MacOS High Sierra 10.13.6 despite adding exec bit as other has suggested. Those who got it to work, please specify what Mac OS version you are using. I wish there is a note about what minimum  MacOS is required to run this.

**Comment by dotWee** [comment]
Just tested the latest release (1.3.0) on macOS Catalina 10.15.7, does not run without manually marking as executable.
